### ISSUE TITLE
Questions about the batch size 4 model

### ISSUE THREAD

**Issue by ypw1996** [clovaai/stargan-v2#40]
Hello, according to the source code you provided,  I keep the same parameters, except batchsize changed to 4, training a model.
      However, it was found that the batch size 4 model  was quite different from the official model, especially in terms of hairstyle and style diversification.
      Could you please help me to analyze the reason? Is it batch size or the parameter used by the official model is different from the source code?
      Thanks a million！

**Comment by yunjey** [comment]
@ypw1996 For stable training, you should use `batch_size >= 8`. We also observed the worse result when using `batch_size=4`.

**Comment by ypw1996** [comment]
Thanks a lot!
Your work is great. I'm interested in it.Hopefully there will be more progress later, such as more precise style control proposed by style GAN.



------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Yunjey Choi"<notifications@github.com&gt;;
发送时间:&nbsp;2020年5月26日(星期二) 下午5:09
收件人:&nbsp;"clovaai/stargan-v2"<stargan-v2@noreply.github.com&gt;;
抄送:&nbsp;"深海鱼"<731357238@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [clovaai/stargan-v2] Questions about the batch size 4 model (#40)





 
@ypw1996 For stable training, you should use batch_size &gt;= 8. We also observed the worse result when using batch_size=4.

**Comment by yunjey** [comment]
@ypw1996 Thanks a lot for your interest!

**Comment by stone100010** [comment]
> Thanks a lot! Your work is great. I'm interested in it.Hopefully there will be more progress later, such as more precise style control proposed by style GAN.
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人:&nbsp;"Yunjey Choi"<notifications@github.com&gt;; 发送时间:&nbsp;2020年5月26日(星期二) 下午5:09 收件人:&nbsp;"clovaai/stargan-v2"<stargan-v2@noreply.github.com&gt;; 抄送:&nbsp;"深海鱼"<731357238@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;; 主题:&nbsp;Re: [clovaai/stargan-v2] Questions about the batch size 4 model (#40) @ypw1996 For stable training, you should use batch_size &gt;= 8. We also observed the worse result when using batch_size=4. — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or unsubscribe.

Please let me know when you have found out. Thank you

**Comment by stone100010** [comment]
You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or unsubscribe.

Please let me know when you have found out. Thank you

**Comment by ypw1996** [comment]
> You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or unsubscribe.
> 
> Please let me know when you have found out. Thank you

Eh, What do you mean?

**Comment by stone100010** [comment]
Hopefully there will be more progress later, such as more precise style control proposed by style GAN.

**Comment by ypw1996** [comment]
> Hopefully there will be more progress later, such as more precise style control proposed by style GAN.

what do you mean?

**Comment by LeonJWH** [comment]
Due to the memory limitation, I set `batch_size=4,` `max_iter=200000`, and also half the learning rate, can I get the same performance with model trained with `batch_size=8`?

**Comment by jokerlc** [comment]
> @ypw1996 For stable training, you should use `batch_size >= 8`. We also observed the worse result when using `batch_size=4`.

so,  if I set the batch_size larger than 8  (as 32)， can get better results?

**Comment by stone100010** [comment]
> > Hopefully there will be more progress later, such as more precise style control proposed by style GAN.
> 
> what do you mean?

like this : https://github.com/woctezuma/stylegan2-projecting-images#3-expression-transfer
